admin-dev/themes/new-theme/js/app/pages/stock/components/overview/products-table.vue

```diff
@@ -26,13 +26,24 @@
   <PSTable class="mt-1">
     <thead>
       <tr class="column-headers">
+        <th
+          scope="col"
+        >
+          <PSSort
+            order="product_id"
+            @sort="sort"
+            :current-sort="currentSort"
+          >
+            {{ trans('title_product_id') }}
```

Comment:
Is "title_product_id" a translatable string?

Reply:
Yep 😊

Reply:
What would be the translation domain? If you show me a screenshot I can tell you the right domain. :-)

Reply:
You've suggested a change, just commented 😊

Reply:
I know :D But the string doesn't seem to be in any domain.

Reply:
@Julievrz 
Mmm I think I don't get you 😊 
The string is this this one: https://github.com/PrestaShop/PrestaShop/pull/28140#discussion_r847275044
And the domain proposed by you is _Admin.Global_, but I'm not sure about that, maybe we can keep _Admin.Catalog.Feature_ or I'd propose _Admin.Catalog.Stock_

Here you are the screen of the table: 
<img width="1678" alt="Schermata 2022-04-12 alle 11 36 10" src="https://user-images.githubusercontent.com/48283112/162930126-4dc373ef-f129-4f17-9df3-308d1f6806e2.png">

Reply:
Sorry, I'll try to rephrase 😇 

The Admin.Catalog.Stock doesn't exist. We could use the Admin.Catalog.Feature (see https://devdocs.prestashop.com/1.7/development/internationalization/translation/translation-domains/). I'm not sure we'll be able to be super specific as this string will probably appear on the product page, which is in Admin.Catalog as well... 😅

But I was talking about this string: 
![image](https://user-images.githubusercontent.com/70583503/162931772-a5d3ae4b-f090-4c44-9590-f7ad753b73b8.png) which doesn't seem to have any domain.

Reply:
@Julievrz
> Sorry, I'll try to rephrase 😇

No worries! Thanks for the document! Didn't aware of it 😊 
And we cannot create a new domain if needed?

> But I was talking about this string:
> ![image](https://user-images.githubusercontent.com/70583503/162931772-a5d3ae4b-f090-4c44-9590-f7ad753b73b8.png) which doesn't seem to have any domain.

That string is a new string, that's why are you saying doesn't seem to have any domain?
The translation and the domain is in the `StockApi.php` file 😊 
I used the _Admin.Catalog.Feature_ but you suggest _Admin.Global_, so you're saying to apply _Admin.Global_ because the string doesn't have any domain yet?

Reply:
Ok, I understand now, I thought we were talking about 2 different strings. 🤯 😄 

The string 'ID' already exists in Admin.Global, that's why I told you to use Admin.Global, to avoid creating a similar string in another domain (because when you do that, that's more work for translators). The string 'ID' can appear on many pages of the back office, that's why using Admin.Global seemed like a good idea: translators will only have to translate it one time for the whole BO. 

However, if you think the translation might be different on the Stock page, you could be more specific and create a new string in Admin.Catalog.Feature.

I hope that helps.

Reply:
Oh ok, got it!
I think it should be better distinguish the translations for each page, but maybe it is very specific case 🤔
The fact is we have a lot of "Entity" and the `ID` is the base field for them, I don't remember in how much _Grid_ we show it, and for someone translating that one will translate also this one (in the Stock page), wdyt?